fbfmaproom/fbfmaproom.py

```diff
@@ -383,6 +383,8 @@ def get_mpoly(mode, country_key, geom_key):
         mpolygon = MultiPolygon([Polygon([(x0, y0), (x0, y1), (x1, y1), (x1, y0)])])
     else:
         label, mpolygon = retrieve_geometry2(country_key, int(mode), geom_key)
+        if isinstance(mpolygon, Polygon):
+            mpolygon = MultiPolygon([mpolygon])
```

Comment:
Do this in retrieve_geometry2, so we don't have to remember to do it every time we call that function.